From 4f18fcb5b2aa00e17d2e96e1af6d0949a5e95fe9 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Mon, 13 Jan 2025 11:24:21 +0100
Subject: [PATCH 1441/1695] plat-stm32mp2: set TEE_RAM_VA_SIZE to 4Mb

Need to increase TEE_RAM_VA_SIZE to be able to compile with
CFG_CC_OPT_LEVEL=0.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 884f956410d4c4e684e7f06389583fc1b7fbd70d plat-stm32mp2: set TEE_RAM_VA_SIZE to 4Mb]
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: If2012716e3c959d259bb4d26bf68f24dbfae916d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/428418
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index ba962b058..3124da656 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -145,6 +145,7 @@ CFG_TZDRAM_SIZE  ?= 0x02000000
 CFG_CORE_LARGE_PHYS_ADDR ?= y
 CFG_CORE_ARM64_PA_BITS ?= 34
 
+CFG_TEE_RAM_VA_SIZE ?= 0x400000
 CFG_CORE_HEAP_SIZE ?= 393216
 CFG_CORE_RESERVED_SHM ?= n
 CFG_DTB_MAX_SIZE ?= 262144
-- 
2.47.2

